CMakeLists.txt

```diff
@@ -292,6 +292,7 @@ add_compile_options(
     #-Wno-newline-eof
     $<$<CXX_COMPILER_ID:GNU>:$<$<COMPILE_LANGUAGE:CXX>:-Wno-class-memaccess>>
     $<$<CXX_COMPILER_ID:GNU>:$<$<COMPILE_LANGUAGE:CXX>:-Wno-interference-size>>
+    $<$<CXX_COMPILER_ID:GNU>:$<$<COMPILE_LANGUAGE:CXX>:-Wno-overloaded-virtual>>
```

Comment:
For now let's not disable it until override keyword is used all over the code base.
```suggestion
```